test/puppetlabs/trapperkeeper/services/metrics/metrics_service_test.clj

```diff
@@ -21,6 +24,15 @@
   ([resp keywordize?]
    (-> resp :body slurp (json/parse-string keywordize?))))
 
+(defn jolokia-encode
```

Comment:
From what I've seen, the need to do this level of encoding comes up fairly rarely. It's required for the tests since they attempt a read operation on every listable MBean and some of the obscure stuff under the Jetty namespace requires a heavy level of escaping.

That being said, it's probably a good idea to add a pointer to the Jolokia escaping rules in our docs as they explain when this is needed and how to avoid it by using POST instead of GET:

https://jolokia.org/reference/html/protocol.html#escape-rules

That being said, it's probably a good idea to